storybook/storybook-react/src/components/grid/PageGrid.stories.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { GridCell, PageGrid, Paragraph } from '@amsterdam/design-system-react/src'
+import { Meta, StoryObj } from '@storybook/react'
+
+import '@amsterdam/design-system-css/src/grid/page-grid.scss'
+import '@amsterdam/design-system-css/src/grid/grid-cell.scss'
+import '@amsterdam/design-system-css/src/heading/heading.scss'
+import '@amsterdam/design-system-css/src/paragraph/paragraph.scss'
+import '../../../../storybook-css/src/docs/stories.scss'
+
+const meta = {
+  title: 'PageGrid',
+  id: 'page-grid',
+  component: PageGrid,
+} satisfies Meta<typeof PageGrid>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
+        {Array.from(Array(12).keys()).map((i) => (
+          <GridCell key={i}>
+            <div className="amsterdam-docs-pink-box">
+              <Paragraph>{i + 1}</Paragraph>
+            </div>
+          </GridCell>
+        ))}
+      </>
+    ),
+  },
+  parameters: {
+    docs: {
+      layout: 'centered',
+    },
+  },
+}
+
+export const ThreeColumns: Story = {
+  args: {
+    children: (
+      <>
```

Comment:
We can lose the fragment here and wrap the `GridCell`s in an array, to improve the Storybook code view. [See the Unordered List stories for an example](https://github.com/Amsterdam/design-system/blob/main/storybook/storybook-react/src/components/UnorderedList/UnorderedList.stories.tsx)